src/Icon/Icon.js

```diff
@@ -31,14 +31,17 @@ export default class Icon extends React.Component {
   };
 
   getLocals() {
+    const { className, icon, color, style = {} } = this.props;
```

Comment:
mmmm ok, but I guarantee you we're doing `getLocals({ ... })` in 90%+ of components in our projects :)